tests/ui/argument-suggestions/extra_arguments.stderr

```diff
@@ -272,9 +271,7 @@ error[E0061]: this function takes 0 arguments but 2 arguments were supplied
   --> $DIR/extra_arguments.rs:14:9
    |
 LL |         empty(1, $y);
-   |         ^^^^^ ----- help: remove the extra argument
-   |               |
-   |               unexpected argument of type `{integer}`
+   |         ^^^^^ - unexpected argument of type `{integer}`
```

Comment:
We should be able to suggest `empty()` here. Should `normalize_span` return an `Result<Span, Span>` , Ok if successfully normalized, Err otherwise, instead of merging both cases?

Reply:
AFAIK there is no way to go from the span of a macro argument

```rust
foo!(~, 1);
        ^ here
```

to the span of the metavariable 

```rust
empty(1, $y);
         ^^ here
```

But it should be possible to suggest removing the other argument that is inside the macro, I'll look into that.

Reply:
Suggesting to remove the argument in macros breaks this test: https://github.com/rust-lang/rust/blob/d1e79b1c74c0b68ef200b317de5aedb8790dc004/tests/ui/macros/issue-26094.rs#L1-L12

Since the error already says `` unexpected argument of type `whatever` `` and that should be enough to figure out a solution, I think it's probably better to just leave the suggestion off.